src/main/java/com/ita/if103java/ims/dao/impl/ItemDaoImpl.java

```diff
@@ -62,7 +62,7 @@ public Item findItemByName(String name, Long accountId) {
         try {
             return jdbcTemplate.queryForObject(Queries.SQL_SELECT_ITEM_BY_NAME, itemRowMapper, accountId, name);
         } catch (EmptyResultDataAccessException e) {
-            throw new ItemNotFoundException("Failed to get item during `select` {name = " + name + "}", e);
+            return null;
```

Comment:
Is this method used anyone else?

I woild left exception, but below handle it correctly, not only empty catch block